nameserver/server_test.go

```diff
@@ -77,6 +78,15 @@ func TestDNSServer(t *testing.T) {
 	if !(len(r.Answer) > 0) {
 		t.Fatal("Number of answers > 0")
 	}
+	// Now ask a query that we expect to return a lot of data.
+	m.SetQuestion("cisco.com.", dns.TypeANY)
+	m.SetEdns0(UDPBufSize, false) // Special magic to coax the other end into complying
```

Comment:
It was my best attempt at forcing a large reply.  I subsequently discovered that doing it this way invalidates the test, and made the above comment.